Rubberduck.Parsing/VBA/RubberduckParserState.cs

```diff
@@ -584,7 +584,8 @@ public void ClearStateCache(VBProject project, bool notifyStateChanged = false)
 
             if (notifyStateChanged)
             {
-                OnStateChanged();
+                OnStateChanged(ParserState.ResolvedDeclarations);   // trigger test explorer and code explorer updates
+                OnStateChanged(ParserState.Ready);   // trigger find all references &c. updates
```

Comment:
We did it this way for speed during a normal parse.  The TE is particularly critical for speed, at least according to Mat.  Yeah, it's a pain here, and I don't like this myself; I'll see if I can't clean this up.